src/util/DefaultValueProcessor.ts

```diff
@@ -0,0 +1,35 @@
+import * as _ from "lodash";
+
+export const addValues = (testCase: any): any => {
+  //create a clone of testCase
+  const resultJson: any = _.clone(testCase);
```

Comment:
```suggestion
  const resultJson: any = _.cloneDeep(testCase);
```
since the object has nested arrays and objects